client/src/components/Reviews.jsx

```diff
@@ -1,5 +1,6 @@
-import React from 'react';
+import React, { useState } from 'react';
```

Comment:
Is it necessary to import useState? I can't find anywhere that you're making use of it, so it's just decreasing performance